bpf/lib/nodeport.h

```diff
@@ -761,8 +761,7 @@ int tail_nodeport_nat_egress_ipv6(struct __ctx_buff *ctx)
 					  info->sec_label,
 					  NOT_VTEP_DST,
 					  (enum trace_reason)CT_NEW,
-					  TRACE_PAYLOAD_LEN,
-					  (int *)&fib_params.l.ifindex);
```

Comment:
Isn't the `fib_params.l.ifindex` value used in the [consequent call](https://github.com/cilium/cilium/pull/24732/files#diff-304969c111195cf4253b336fc67c9891f17b91d49738a0cc334333ac11e84b9eL811) to `fib_redirect`?

Reply:
Ok, I see, there's `use_tunnel` flag below

Reply:
exactly - maybe not so trivial after all, sorry :smile:. But even better to clean it up then, and clarify how that struct is used ...